src/classes/CON_Email.cls

```diff
@@ -233,22 +233,21 @@ public class CON_Email {
     * @param api The API name of the email field
     */
     private class CON_EmailField {
-
         public String value {get;set;}
-        public String label {get;set;}
+        public String lbl {get;set;}
         public String apiName {get;set;}
         public String prefLabel {get;set;}
 
         public CON_EmailField(String val, String lbl, String api ) {
             // Store default email field mappings to preferred email pick list.
             Map<String,String> preferredEmailMappings = new Map<String,String>{
-                'University Email' => 'University',
-                'Work Email' => 'Work',
-                'Alternate Email' => 'Alternate'
```

Comment:
I am not sure about this one. It looks like if this map does not contain some Spanish word then it will just use the Spanish word itself. Is it true that this functionality is only used for English?

Reply:
This is pretty much only used for English, but even further, it exists only because the default values of "Preferred Email" in HEDA dont match the labels of the different email fields. So this little map only exists to support the packaged values of "Preferred Email". If an admin changes the values in "Preferred Email" those updates will still need to match an existing email field.